.github/workflows/benchmark.yml

```
@@ -0,0 +1,74 @@
+name: run-benchmarks
+on:
+  workflow_dispatch:
+    inputs:
+      sha:
+        description: 'SHA of the commit to benchmark'
+        required: true
+
+permissions:
+  contents: read
+  packages: read
+
+jobs:
+  http-benchmark:
+    runs-on: ubuntu-22.04
+    permissions:
+      contents: read
+      packages: read
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+
+      - name: Install k6
+        run: |
+          sudo gpg -k
+          sudo gpg --no-default-keyring --keyring /usr/share/keyrings/k6-archive-keyring.gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys C5AD17C747E3415A3642D57D77C6C491D6AC1D69
+          echo "deb [signed-by=/usr/share/keyrings/k6-archive-keyring.gpg] https://dl.k6.io/deb stable main" | sudo tee /etc/apt/sources.list.d/k6.list
+          sudo apt-get update
+          sudo apt-get install k6
+
+      - name: Start services
+        run: |
+          sudo docker compose -f benches/http-loadtesting/docker-compose.yml up -d
+      - name:  Install wash
+        run: |
+          curl -s https://packagecloud.io/install/repositories/wasmcloud/core/script.deb.sh | sudo bash
+          sudo apt-get install wash
```

```suggestion
      - name:  Install wash
        uses: wasmcloud/common-actions/install-wash@main
```

supernit, this is a little action we have prepared